docs/alma/alma.rst

```diff
@@ -124,84 +123,98 @@ Querying Targets and Regions
-    'science_keyword', 'scientific_category', 'lastModified']
-
-
-Please note that some of the column names are duplicated. First group of names
-(the ones containing "_") are column names as they appear in the ALMA ObsCore
-model while the second group are copies created to maintain backwards
-compatibility with previous version of the library.
+    'obs_release_date', 'spatial_resolution', 'frequency_support', 'frequency',
+    'velocity_resolution', 'obs_creator_name', 'pub_title', 'first_author',
+    'authors', 'pub_abstract', 'publication_year', 'proposal_abstract',
+    'schedblock_name', 'proposal_authors', 'sensitivity_10kms',
+    'cont_sensitivity_bandwidth', 'pwv', 'group_ous_uid', 'member_ous_uid',
+    'asdm_uid', 'obs_title', 'type', 'scan_intent', 'science_observation',
+    'spatial_scale_max', 'qa2_passed', 'bib_reference', 'science_keyword',
+    'scientific_category', 'lastModified']
 
 
 Region queries are just like any other in astroquery:
 
-
 .. code-block:: python
-
+.. doctest-remote-data::
     >>> from astropy import coordinates
     >>> from astropy import units as u
     >>> galactic_center = coordinates.SkyCoord(0*u.deg, 0*u.deg,
     ...                                        frame='galactic')
     >>> gc_data = Alma.query_region(galactic_center, 1*u.deg)
-    >>> print(len(gc_data))
-    383
+    >>> print(gc_data) # doctest: +IGNORE_OUTPUT
+         obs_publisher_did      obs_collection facility_name ...     scientific_category           lastModified
+                                                             ...
+    --------------------------- -------------- ------------- ... --------------------------- -----------------------
+    ADS/JAO.ALMA#2012.1.00133.S           ALMA           JAO ...      ISM and star formation 2021-09-30T16:34:41.133
 
 Querying by other parameters
 ============================
 
 As of version 0.3.4, you can also query other fields by keyword. For example,
-if you want to find all projects with a particular PI, you could do:
+if you want to find all projects in a region with a particular PI, you could do:
+
+.. code-block:: python
+
+    >>> rslt = Alma.query_region('W51', radius=25*u.arcmin, pi_name='*Ginsburg*') # doctest: +REMOTE_DATA
+
+or if you wanted all projects by a given PI:
 
 .. code-block:: python
 
-   >>> rslt = Alma.query_object('W51', pi_name='*Ginsburg*', public=False)
+   >>> rslt = Alma.query(payload=dict(pi_name='Ginsburg, Adam')) # doctest: +REMOTE_DATA
 
 The ''query_sia'' method offers another way to query ALMA using the IVOA SIA
-subset of keywords returning results in 'ObsCore' format.
+subset of keywords returning results in 'ObsCore' format.  For example,
+to query for all images that have `XX` polarization (note that this query is too large
+to run, it is just shown as an example):
 
 .. code-block:: python
 
-    >>> Alma.query_sia(query_sia(pol='XX'))
+    >>> Alma.query_sia(pol='XX') # doctest: +SKIP
```

Comment:
This example times out for me, so I tried a simpler one and got an error:
https://github.com/astropy/astroquery/issues/2240